doc/about/about.md

```diff
@@ -42,19 +43,19 @@ For example, the following URI schemes are used in the documentation:
+-   The **scheme** describes how to connect and is sometimes referred to as the protocol. When specifying URIs in a Gateway configuration, you can use `tcp://{hostname}:{port}` to make a basic TCP connection, or specify any of the supported schemes such as http, https, ws, wss, sse, and so on. See the [supported URI schemes](../admin-reference/r_configure_gateway_service#note_supportedURLschemes) for the complete list.
 -   The **host** specifies where to connect and can be a hostname or domain name, or an IP address.
 -   The **port** specifies the port number to ask for. This portion of the URI scheme is optional if you are using a default port, such as port 80 for http or port 443 for https. For example, when using the http scheme you do not need to specify port 80.
 -   The **path** refers to the path of the resource. At a minimum you must specify the root path (`/`). Thus, `http://example.com/` is a legal address, but `http://example.com` is not, even though in practice the final slash "`/`” is added automatically.
 
-In addition, you can append a query string to the URL to provide non-hierarchical information. The query string follows a question mark (?) appended to the URL. For example, you can use this query string when configuring security for your Gateway (as described in the [Security Reference](../admin-reference/r_conf_security.md)).
+In addition, you can append a query string to the URL to provide non-hierarchical information. The query string follows a question mark (?) appended to the URL. For example, you can use this query string when configuring security for your Gateway (as described in the [Security Reference](../admin-reference/r_configure_gateway_security.md)).
 
 **Note:** In the example URLs, the `${gateway.hostname}` syntax allows you to define property values once and then the values are propagated throughout the configuration when the Gateway starts. You can replace any value in the configuration file that uses the dollar-sign and curly brace format (such as `${gateway.hostname}`) with a property. In the Gateway configuration, you can configure property defaults such as `gateway.hostname`, `gateway.base.port`, `gateway.extras.port`.
-### See Also
 
--   [Service Reference](../admin-reference/r_conf_service.md) for information about specifying URLs with the accept and connect elements.
--   Wikipedia description of [URI Scheme](http://en.wikipedia.org/wiki/URI_scheme)
--   [Configuring Multiple Services on the Same Host and Port](../admin-reference/c_conf_multipleservices.md#configmultsrvcs)
+About Domain Names
+----------------------------------------
+Many examples in the documentation use the `.net domain` (such as `tcp://gateway.example.net:8080`) to indicate internal, nonpublic URLs, and use the `.com` domain to indicate public URLs. All domains and URLs are for example purposes only. Simply replace any instances of "example.com" in the configuration with your domain or hostname.
```

Comment:
"Simply replace any instances of "example.com" in the configuration with your domain or hostname."

I think this should say something more like, "Replace hostnames and domains used in the documentation with your own hostnames or domains." Otherwise the sentence is too specific.